### PR TITLE
feat: add radio buttons

### PR DESCRIPTION
**Pull request by korotkovoleksii**

### CODE REVIEW

File: src/components/CreatePollForm.vue [hunk 1]
```diff
@@ -91,12 +119,13 @@ import Textarea from 'primevue/textarea';
...
 import Calendar from 'primevue/calendar';
 import InputText from 'primevue/inputtext';
+import RadioButton from 'primevue/radiobutton';
 import useVuelidate from '@vuelidate/core';
 import { pollValidations } from '@/utils/poll-validations';
 import { StoreModuleEnum } from '@/store/types';
 import { HousesActionsEnum, HousesGettersEnum } from '@/houses/store/types';
 import { HouseModel } from '@/houses/models/house.model';
-import { PollsActionEnum } from '@/store/polls/types';
+import { PollsActionEnum, PollAcceptanceCriteria } from '@/store/polls/types';
```
Comment (aleonovdementiev) [inline src/components/CreatePollForm.vue]:
```suggestion
import { PollsActionEnum, PollAcceptanceCriteriaEnum } from '@/store/polls/types';
```

File: src/components/CreatePollForm.vue [hunk 2]
```diff
@@ -114,20 +144,28 @@ export default defineComponent({
     },
   },
   data() {
+    const pollAcceptanceCriteriaEnum = Object.keys(PollAcceptanceCriteria).map((name) => {
```
Comment (aleonovdementiev) [inline src/components/CreatePollForm.vue]: Why do you need this variable?

File: src/store/polls/index.ts [hunk 3]
```diff
@@ -16,6 +16,7 @@ export const state: PollsStateInterface = {
     polledHouses: [],
     status: 'active',
     type: '',
+    acceptanceCriteria: '',
```
Comment (aleonovdementiev) [inline src/store/polls/index.ts]: Is there an option, when acceptanceCriteria can be empty?

File: src/components/CreatePollForm.vue (hunk 2 above)
Comment (korotkovoleksii) [reply src/components/CreatePollForm.vue]: For getting  array of objects (name, value) from enum

File: src/store/polls/index.ts (hunk 3 above)
Comment (korotkovoleksii) [reply src/store/polls/index.ts]: acceptanceCriteria can be empty when user doesn't select poll

**Review by aleonovdementiev — APPROVED** [review]

**Review by yevhendevbox — APPROVED** [review]

**Pull request merged**